sagemaker_neo_compilation_jobs/mxnet_mnist/mxnet_mnist_neo.ipynb

```diff
@@ -212,9 +208,7 @@
     "\n",
     "Now that our Endpoint is deployed and we have a ``predictor`` object, we can use it to classify handwritten digits.\n",
     "\n",
-    "To see inference in action, draw a digit in the image box below. The pixel data from your drawing will be loaded into a ``data`` variable in this notebook. \n",
-    "\n",
-    "**Note:** After drawing the image, you'll need to move to the next notebook cell."
+    "To see inference in action, we define a variable `data` and initialize it with the binary equivalent of a hand drawn digit `0`"
```

Comment:
we can refer to this other python script here, saying how we got this input data

Reply:
Thanks for taking this up offline. We are now good to merge it.